boards/beagle/beaglebone_ai64/beaglebone_ai64_j721e_main_r5f0_0.yaml

```diff
@@ -14,4 +14,5 @@ toolchain:
   - xtools
 supported:
   - uart
+  - i2c
 vendor: beagle
```

Comment:
@malto101 Can you please update in https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/beagle/beaglebone_ai64/doc/index.rst#supported-features also?

Reply:
I Have updated the docs to show the same.